js-raphtory/src/graph/graph_view_impl.rs

```diff
@@ -2,9 +2,9 @@ use super::{Graph, UnderGraph};
 use raphtory::db::api::view::{Base, BoxableGraphView, InheritViewOps};
 
 impl Base for Graph {
-    type Base = dyn BoxableGraphView<'static> + Send + Sync + 'static;
+    type Base = dyn BoxableGraphView + Send + Sync + 'static;
 
-    fn base(&self) -> &(dyn BoxableGraphView<'static> + Send + Sync + 'static) {
+    fn base(&self) -> &(dyn BoxableGraphView + Send + Sync + 'static) {
```

Comment:
meaning?
```rust
fn base(&self) -> &Self::Base
```

Reply:
The lifetime for `BoxableGraphView` is no longer needed because it is not dealing with the iterators any more.

Reply:
what I meant is you can just use Self::Base instead of copying the type that is right above